stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -4559,12 +4678,14 @@ impl SortitionDB {
         Ok(ret)
     }
 
+    /// DO NOT CALL during Stacks block processing (including during Clarity VM evaluation). This function returns the latest data known to the node, which may not have been at the time of original block assembly.
```

Comment:
This is just for my own knowledge, not a nit: I am not entirely sure I understand what this means? Could you explain a tiny bit more for me?

Reply:
Ah that makes sense! thank you :D